waiter/config-full.edn

```diff
@@ -589,7 +589,11 @@
 
           ;; Waiter aggregates metrics locally and publishes
           ;; to StatsD on the following interval (milliseconds):
-          :publish-interval-ms 10000}
+          :publish-interval-ms 10000
+
+          ;; Waiter aggregates metrics locally for instances and resource usage and
+          ;; maintains them locally on the following interval (milliseconds):
```

Comment:
How about "Waiter polls the router state for instances and resource usage on the following interval (milliseconds):"?

Reply:
Changed.